ostelco-ios-client/Network/UserManager.swift

```diff
@@ -7,35 +7,139 @@
 //
 
 import Crashlytics
+import FirebaseAuth
 import ostelco_core
+import PromiseKit
 
-class UserManager {
+class UserManager: NSObject {
```

Comment:
this probably doesn't still need to be an `NSObject` subclass since it's not a firebase UI delegate anymore